index.html

```diff
@@ -0,0 +1,243 @@
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me..</p>
+                    <div class="orange-button2">
+                        <div class="see-project2">Get my resume</div>
+                    </div>                    
+                </div>            
+                <hr>
+
+            <ul>
+                <li class="h3">Languaje</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+
+                <li class="h3">Framework</li>
+                <li>React</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+            
+                <li class="h3">Skills</li>
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API design</li>
+                <li>Web Development</li>
+            </ul>
+           </section> 
+            
+            <section id="contact">            
+            
+                <h2><b> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</b></h2>
+
+                <div class="flex-container">
+                    <form class="form">
+                        <div class="form-input">
+                            <input type="text" placeholder="Full name" class="form-input">
+                        </div>
+                        <div class="form-input">
+                            <input type="email"  placeholder="Email" class="form-input"> 
+                        </div>
+                        <div class="form-section">
+                            <textarea class="form-box"  placeholder="Write something here..."></textarea>                    
+                        </div>
+                        
+                        <div class="send-button">
+                            <button class="send">Get in touch</button>
+                        </div>
+                    </form>
+                </div>
+                
+                <hr>
+
+                <div class="social-media">
+                    <a href="">
+                    <img src="Assets/GitB.svg" alt="Github">
+                    </a>
+                    <a href="">
+                    <img src="Assets/LinB.svg" alt="Linkedin">
+                    </a>
+                    <a href="">
+                    <img src="Assets/AngelisB.svg" alt="Angelist">
+                    </a>
+                    <a href="">
+                    <img src="Assets/TwitterB.svg" alt="Twitter">
+                    </a>
+                    <a href="">
+                    <img src="Assets/MediumB.svg" alt="Medium">
+                    </a>
+                </div>
+        </section>
```

Comment:
- [ ] Please remove this line of code because it is not part of this milestone and needs to be added in the next milestone where you have to add this thing again with valid commits. Always try to do things in a systematic manner that will help you to be more focused on a particular thing with good learning as an outcome.